Pipeline: filters: nightfall: style updates (#1694)
fluent/fluent-bit-docs: pipeline/filters/nightfall.md

# Nightfall

The _Nightfall_ filter scans logs for sensitive data and redacts the sensitive portions. This filter supports scanning for
various sensitive information, ranging from API keys and Personally Identifiable Information (PII) to custom regular expressions you
define. You can configure what to scan for in the [Nightfall Dashboard](https://app.nightfall.ai).

> This filter isn't enabled by default in 1.9.0 due to a typo. To enable it, set the flag ```-DFLB_FILTER_NIGHTFALL=ON``` when building. In 1.9.1 and above this is fixed.

Reviewer: ```suggestion
This filter isn't enabled by default in version 1.9.0 due to a typo. To enable it,
set the flag ```-DFLB_FILTER_NIGHTFALL=ON``` when building. This is fixed for
versions 1.9.1 and later.
```

## Configuration parameters

The plugin supports the following configuration parameters:

| Key | Description | Default |
| :--- | :--- | :--- |
| `nightfall_api_key` | The Nightfall API key to scan your logs with, obtainable from the [Nightfall Dashboard](https://app.nightfall.ai) | _none_ |
| `policy_id` | The Nightfall developer platform policy to scan your logs with, configurable in the [Nightfall Dashboard](https://app.nightfall.ai/developer-platform/policies). | _none_ |
| `sampling_rate` | The rate controlling how much of your logs you wish to be scanned. Must be a float between `(0,1]`. `1` means all logs will be scanned. Use this setting to avoid rate limits in conjunction with Fluent Bit match rules.| `1` |
| `tls.debug` | Debug level between `0` (nothing) and `4` (every detail). | `0` |
| `tls.verify` | When enabled, turns on certificate validation when connecting to the Nightfall API. | `true` |
| `tls.ca_path` | Absolute path to root certificates, required if `tls.verify` is true. | _none_ |

### Configuration file

> The following is an example of a configuration file for the Nightfall filter.

Reviewer: ```suggestion
The following is an example of a configuration file for the Nightfall filter:
```

    Name stdout
```

### Command line

> Once configured, you can use the filter from the command line by running a command like:

Reviewer: ```suggestion
After you configure the filter, you can use the it from the command line by running a
command like:
```

> ```shell

Author: We don't know that gitbook supports this
